debian/changelog

```diff
@@ -1,3 +1,72 @@
+snapcraft (2.29) xenial; urgency=medium
+
+  [ Sergio Schvezov ]
+  * store: better retry strategy for GETs. (#1212)
+  * docs: remove docs and link to snappy-docs. (#1227)
+  * nodejs plugin: switch to the newer LTS. (#1228)
+  * pluginhandler: exclude `/snap/` from libraries. (#1231)
+  * nodejs plugin: switch to the newer LTS. (#1228) (#1232)
```

Comment:
That (#1228) should probably be removed. Maybe you should also remove the previous entry about switching to the newer LTS.